llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -525,17 +524,19 @@ void SpirvLowerGlobal::handleStoreInstGlobal(StoreInst &storeInst) {
 // @param storeInst : Store instruction
 void SpirvLowerGlobal::handleStoreInstGEP(GetElementPtrInst *const getElemPtr, StoreInst &storeInst) {
   Value *storeValue = storeInst.getOperand(0);
+
   std::vector<Value *> indexOperands;
 
+  assert(cast<ConstantInt>(getElemPtr->getOperand(1))->isZero() && "Non-zero GEP first index\n");
```

Comment:
Is the operand list zero-indexed? If so, "first index" might not be the right assertion failure message.

Reply:
Yes, we expect the first index operand (the second operand of GEP) is always 0 to dereference the GEP pointer